Chap_API_Reserved_Keys.tex

```diff
@@ -193,32 +193,38 @@ \subsection{Session realm attributes}
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \subsection{Job realm attributes}
-\label{chap:res:jrealm}
-
-Job-related information is retrieved by including the namespace of the target
-job and a rank of \refconst{PMIX_RANK_WILDCARD} in the \refarg{proc} argument
-passed to \refapi{PMIx_Get}. If desired for code clarity, the caller can also
-include the \refattr{PMIX_JOB_INFO} attribute in the \refarg{info} array,
-though this is not required. If information is requested about a namespace in
+\label{chap:api_rsvd_keys:jrealm}
+
+Job-related information can be retrieved by requesting a key which defaults
+to the job realm or by including the \refattr{PMIX_JOB_INFO} attribute
+in the \refarg{info} array passed to \refapi{PMIx_Get}.
+For job-related keys the target job is specified by setting the namespace of the target
+job in the \refarg{proc} argument and specifying a rank of \refconst{PMIX_RANK_WILDCARD} 
+in the \refarg{proc} argument passed to \refapi{PMIx_Get}.
+
+% Removing because it is only true until we add a key that defaults to another
+% realm, but can be used for job realms.  Then this attribute becomes required.
+%If desired for code clarity, the caller can also
+%include the \refattr{PMIX_JOB_INFO} attribute in the \refarg{info} array,
+%though this is not required.
+
+If information is requested about a namespace in
 a session other than the one containing the requesting process, then the
 \refarg{info} array must contain a \refattr{PMIX_SESSION_ID} attribute
 identifying the desired target session. This is required as
 many environments only guarantee unique namespaces within a session, and not
 across sessions.
 
-Job-level information includes the following attributes:
+The following keys, by default, request job-level information:
+They will return information about the job indicated in \refarg{proc}:
 
-%
-\declareAttribute{PMIX_NSPACE}{"pmix.nspace"}{char*}{
```

Comment:
Did this get moved somewhere else? It is required and used in various places - e.g., to retrieve my namespace when someone doesn't want to globally store the `pmix_proc_t` returned by `PMIx_Init`

Reply:
I see this was moved to the process realm - no issues with doing so as long as it is somewhere.